Update Fork (#25)
hvanyo/Bodiless-JS: packages/cx-flowcontainer/doc/FlowContainer.md

# Vital Flow Container Tokens

> The Vital Flow Container is a passthrough, and uses the [Bodiless Flow

Reviewer: ```suggestion
The Vital Flow Container is built on top of the [Bodiless Flow
```

> The Vital Flow Container is just a passthrough, and uses the [Bodiless Flow

Author: @misterjones  should we remove "just" here?   I often put them it to make it sound simple and then remove because I believe its little like talking down to reader?

Reviewer 2: I agree, I try to remove those as well. This one slipped through!

> Container](/Components/FlowContainer/). The Vital Flow Container package does, however, provide some

Reviewer: ```suggestion
Container](/Components/FlowContainer/). It provides a basic set of components to choose from, along with some
```

## Site Builder Details

### Usage

> ```js

Author: @misterjones  i would remove example without context

Reviewer 2: @hvanyo Ha, you beat me to it. 😆 

I was going to ask if this was still a useful example, and, if so, if you could help me with describing what it does.

      ComponentWrapper: cxSpacing.Gutter,
    },
  },

>   WithComponentsVariations, // a token that would define list of components available for this FlowContainer

Reviewer: This otken does not exist.  Also we have not documented all available tokens - see [here](https://github.com/johnsonandjohnson/Bodiless-JS/blob/main/packages/cx-flowcontainer/src/components/FlowContainer/tokens/cxFlowContainer.ts#L158-L168)

Author: @misterjones  let me know if need assistance here...

Reviewer 2: We discussed removing the token descriptions, and allowing them to be documented in the API documentation, correct?
